deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -264,6 +257,17 @@ spec:
           - list
           - update
           - watch
+        - apiGroups:
+          - ceph.rook.io
+          resources:
+          - cephfilesystemsubvolumegroups
+          verbs:
+          - create
+          - get
+          - list
+          - patch
+          - update
+          - watch
```

Comment:
This seems unrelated to this PR and should not be coupled with it

Reply:
This is actually due to the removal of the storagerequest controller file, See here https://github.com/red-hat-storage/ocs-operator/pull/3070/files#diff-8eed42ca79a1fa01b006a52bf1f622ed8f37ca079fe3eee8b9cb6655a08e2892L77